survey/dashboards/constants.py

```diff
@@ -0,0 +1,125 @@
+"""
+Constants settings for the survey dashboard
+
+This script defines the visual styling, colour palettes, maturity levels, and other
+constants used throughout the dashboard.
+"""
+
+THEME = {
+    "primary": "#1192e8",
+    "primary_bg": "#e5f6ff",
+    "btn_primary": "#0072c3",
+    "text_color": "#393939",
+    "font_family": "Source Sans Pro, sans-serif",
+    "bg_color": "#f8f9fa",
+    "agree_color": "#198038",
+}
+
+# Colour palette and maturity levels
+COLOUR_PALETTE = [
```

Comment:
Some of these constants should probably `tuple`s because they want to be immutable.